releng/org.eclipse.xtext.target/org.eclipse.xtext.target.target

```diff
@@ -1,99 +1,92 @@
+<unit id="org.eclipse.xtext.purexbase.ide" version="0.0.0"/>
+<unit id="org.eclipse.xtext.purexbase.ide.source" version="0.0.0"/>
+<unit id="org.eclipse.xtext.purexbase.source" version="0.0.0"/>
+<unit id="org.eclipse.xtext.smap" version="0.0.0"/>
+<unit id="org.eclipse.xtext.smap.source" version="0.0.0"/>
+<unit id="org.eclipse.xtext.xbase" version="0.0.0"/>
+<unit id="org.eclipse.xtext.xbase.ide" version="0.0.0"/>
+<unit id="org.eclipse.xtext.xbase.ide.source" version="0.0.0"/>
+<unit id="org.eclipse.xtext.xbase.source" version="0.0.0"/>
+<unit id="org.eclipse.xtext.xbase.testing" version="0.0.0"/>
+<unit id="org.eclipse.xtext.xbase.testing.source" version="0.0.0"/>
+<unit id="org.eclipse.xtext.xbase.testlanguages" version="0.0.0"/>
+<unit id="org.eclipse.xtext.xbase.testlanguages.ide" version="0.0.0"/>
+<unit id="org.eclipse.xtext.xbase.testlanguages.ide.source" version="0.0.0"/>
+<unit id="org.eclipse.xtext.xbase.testlanguages.source" version="0.0.0"/>
+<repository location="http://services.typefox.io/open-source/jenkins/job/xtext-extras/job/master/lastStableBuild/artifact/build/p2-repository/"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="com.google.guava" version="0.0.0"/>
+<unit id="org.eclipse.draw2d.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.emf.mwe2.lib" version="0.0.0"/>
+<unit id="org.eclipse.emf.sdk.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.jdt.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.m2e.feature.feature.group" version="1.5.1.20150109-1820"/>
+<unit id="org.eclipse.pde.api.tools.ee.feature.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.pde.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.platform.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.xpand" version="0.0.0"/>
+<unit id="org.eclipse.xtend" version="0.0.0"/>
+<unit id="org.eclipse.xtend.typesystem.emf" version="0.0.0"/>
+<repository location="http://download.eclipse.org/releases/luna/201502271000/"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.emf.mwe2.launcher.feature.group" version="0.0.0"/>
+<repository location="http://download.eclipse.org/modeling/emft/mwe/updates/releases/2.7.1/"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="de.itemis.xtext.antlr.sdk.feature.group" version="0.0.0"/>
+<repository location="http://download.itemis.com/updates/releases/2.1.1/"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.junit" version="0.0.0"/>
+<repository location="http://download.eclipse.org/modeling/tmf/xtext/updates/orbit/"/>
+</location>
```

Comment:
This looks like you used the target platform editor to save this file. Unfortunately, this kills formatting and makes tracking changes impossible. Could you change the XML by hand so that it becomes obvious what changed? 
Sorry for being picky on this but I had cases where the target platform editor damaged my files.

Reply:
I had serious problems loading the TP. Finally it works. No changes needed, so I reverted that file.